docs/kola/external-tests.md

```diff
@@ -103,6 +103,68 @@ it out.
 (Previously the API for this was to send `SIGTERM` to the current process; that
 method is deprecated and will be removed at some point)
 
+## HTTP Server
+
+The `kolet` binary is copied into the `core` user's home directory
+(`/var/home/core`) on the CoreOS system running the tests. Notably, it contains
+the built-in command `kolet httpd` for starting an HTTP file server to serve the
+contents of the file system.
+By default, it starts the server at port `8080` and serves the contents of the
+file system at `./`; you can use the `--port` and `--path` flags to override the
+defaults.
+
+For example, if you're using a bash script as your test, you can start an HTTP
+server to serve the contents at `/var/home/core` like this:
+```
+echo testdata > /var/home/core/testdata.txt
+kolet httpd --path /var/home/core/ &
```

Comment:
ohh, this sounds better, thanks!

Reply:
This doesn't really matter too much here, but it's different because the use of `systemd-run` in that way "escapes" the `kola-runext.service` lifecycle.  This relates to https://github.com/coreos/rpm-ostree/pull/2657 too.


It looks like `--slice-inherit` may be cleanest for this?  But it's not in RHEL8 systemd :cry: